examples/editable-layers/editor/package.json

```
@@ -1,4 +1,5 @@
 {
+  "name": "@deck.gl-community/editor",
```

While it wouldn't really matter here, lets only use the npm organization scope for actual modules
```suggestion
  "name": "editable-layers/editor",
```

---

Reverted file